CHANGELOG.md

```diff
@@ -175,6 +202,19 @@ BUG FIXES:
 * ui: Scale down logger height in the UI when the sidebar container also has task events [[GH-15759](https://github.com/hashicorp/nomad/issues/15759)]
 * volumes: Fixed a bug where `per_alloc` was allowed for volume blocks on system and sysbatch jobs, which do not have an allocation index [[GH-16030](https://github.com/hashicorp/nomad/issues/16030)]
 
+## 1.4.7 (March 21, 2023)
+
+IMPROVEMENTS:
+
+* build: Update to go1.20.2 [[GH-16427](https://github.com/hashicorp/nomad/issues/16427)]
+*
+* BUG FIXES:
+*
+* * client: Fixed a bug where clients using Consul discovery to join the cluster would get permission denied errors [[GH-16490](https://github.com/hashicorp/nomad/issues/16490)]
+* * client: Fixed a bug where cpuset initialization fails after Client restart [[GH-16467](https://github.com/hashicorp/nomad/issues/16467)]
+* * plugin: Add missing fields to `TaskConfig` so they can be accessed by external task drivers [[GH-16434](https://github.com/hashicorp/nomad/issues/16434)]
+* * services: Fixed a bug where a service would be deregistered twice [[GH-16289](https://github.com/hashicorp/nomad/issues/16289)]
```

Comment:
I think your editor may have added more `*` than needed. Here's the rendered version:
https://github.com/hashicorp/nomad/blob/7177b9a7db3f4226ab9f7b88773c35374b03b0a9/CHANGELOG.md#147-march-21-2023

Reply:
Good catch! Sorry about that.

Reply:
No worries, I'm always annoyed that this is still not an automated process 😅